### PR TITLE
add new component from lms

### PR DESCRIPTION
**Pull request by alvyfz**

### CODE REVIEW

File: src/stories/sectionModal.stories.tsx
Comment (mugeki) [inline src/stories/sectionModal.stories.tsx:1]: rename to `SectionModal.stories.tsx`

File: yarn.lock
Comment (mugeki) [inline yarn.lock:1]: add `yarn.lock` to `.gitignore`

**Review by mugeki — CHANGES_REQUESTED** [review]
add `index.tsx` file yang isinya mengexport setiap component

File: yarn.lock
Comment (alvyfz) [reply yarn.lock:1]: done

File: src/stories/sectionModal.stories.tsx
Comment (alvyfz) [reply src/stories/sectionModal.stories.tsx:1]: done

**Review by mugeki — APPROVED** [review]
LGTM

**Pull request merged**